jetty-documentation/src/main/asciidoc/quick-start/configuring/how-to-configure.adoc

```diff
@@ -65,10 +65,15 @@ The `*.mod` files are normally located in `$JETTY_HOME/modules/`, but extra or e
 If module changes are required, it is best practice to copy the particular `*.mod` file from `$JETTY_HOME/modules/` to `$JETTY_BASE/modules/` before being modified.
 
 XML files::
-  XML files in link:#jetty-xml-syntax[Jetty IoC XML format] or Spring IoC format are listed either on the command line, in `ini` files or are added to the effective command line by a module definition. 
-  The XML files instantiate and inject the actual Java objects that comprise the server, connectors and contexts. Because Jetty IoC XML files use properties, most common configuration tasks can be accomplished without editing these XML files and can instead be achieved by editing the property in the corresponding `ini` files.
+  XML files in link:#jetty-xml-syntax[Jetty IoC XML format] or Spring IoC format are listed either on the command line, in `ini` files, or are added to the effective command line by a module definition. 
+  The XML files instantiate and inject the actual Java objects that comprise the server, connectors and contexts. 
+  Because Jetty IoC XML files use properties, most common configuration tasks can be accomplished without editing these XML files and can instead be achieved by editing the property in the corresponding `ini` files.
   XML files are normally located in `$JETTY_HOME/etc/`, but extra or edited XML files may be added to `$JETTY_BASE/etc/`. 
   If XML configuration changes are required, it is best practice to copy the XML file from `$JETTY_HOME/etc/` to `$JETTY_BASE/etc/` before being modified.
+  
+Below is an illustration of how the various Jetty configuration files (`ini`, `mod` and XML) are related:
+
+image:images/Jetty_Configuration_File_Relationships.png[image,width=693]
```

Comment:
This image doesn't really capture the relationships.

An ini file may contain:
- module enables
- properties set either name=value or via a foo.properties file
- jar files for the class path (rarely used)
- xml files (rarely used)

A module may contain:
- module dependencies
- jar files for the class path (best place for jars)
- xml files (best place)
- a template to create an ini file for the module